.github/workflows/update-gradle-wrapper.yml

```diff
@@ -8,7 +8,7 @@ on:
 jobs:
   update-gradle-wrapper:
     runs-on: ubuntu-latest
-
+    if: ${{ !github.event.repository.fork && github.ref == 'refs/heads/main' }}
```

Comment:
By the way, `github.ref == 'refs/heads/main'` this was not correct. Now the CI just skip it on regular PRs. I'm going to revert it

Reply:
Not really, I was just tired and couldn't think clearly. 

The goal was:
1) Not to run from a fork
2) Not ro tun from a PR coming from a fork (basically because secrets are not getting passed to to run and still fails)

This finally did it (I hope)
`!github.event.repository.fork && !github.event.pull_request.head.repo.fork`
https://github.com/gmazzo/gradle-buildconfig-plugin/pull/305/files#diff-8ed13df95e3667964f78d1711225fae3857dfa8a46481166740dd4a28920331fR75